controllers/mentor.go

```diff
@@ -9,11 +9,12 @@ import (
 	"strings"
 
 	"kwoc20-backend/models"
+	logs "kwoc20-backend/utils/logs/pkg"
 
 	"github.com/jinzhu/gorm"
 )
 
-// MentorOauth Handler for Github OAuth of Mentor
+// MentorOAuth Handler for Github OAuth of Mentor
```

Comment:
This might be unrelated to this PR, but PLEASE DON'T SKIP ERRORS!
If some error happens with these functions, it would become pretty difficult, impossible to fix them, because the error is hidden.
For eg - 
```
mentorOAuth, _ := mentorOAuth1.(map[string]interface{})
```
Common pattern for this is that, if you have an error, either you can - 
* handle them using a function
* log them, so that if that happens, others can make a fix.
Please, Please don't skip errors!!
cc @grapheo12 @rakaar

Reply:
cool !
wil make an issue